bumble/transport/android_netsim.py

```diff
@@ -318,10 +319,16 @@ def __init__(self, name, manufacturer, hci_device):
             self.hci_device = hci_device
 
         async def start(self):  # Send the startup info
-            chip_info = ChipInfo(
+            device_info = DeviceInfo(
                 name=self.name,
-                chip=Chip(kind=ChipKind.BLUETOOTH, manufacturer=self.manufacturer),
+                kind='Bumble',
```

Comment:
nit: Please update kind to `BUMBLE` to match other device kinds which are all capitalized